### PR TITLE
chore: upgrade urfave-cli v1 to v2

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Migrate all v1 code in the urfave-cli to v2.

### CODE REVIEW

File: components/node/cmd/batch_decoder/main.go [hunk 1]
```diff
@@ -89,17 +89,17 @@ func main() {
 			Name:  "reassemble",
 			Usage: "Reassembles channels from fetched batches",
 			Flags: []cli.Flag{
-				cli.StringFlag{
+				&cli.StringFlag{
```
Comment (Pangssu) [inline components/node/cmd/batch_decoder/main.go:92]: Please change the default value to `0xff00000000000000000000000000000000000255`.

File: components/validator/flags/flags.go [hunk 2]
```diff
@@ -3,120 +3,123 @@ package flags
...
+	GuardianEnabledFlag = &cli.BoolFlag{
+		Name:    "guardian.enabled",
+		Usage:   "Enable guardian",
+		EnvVars: prefixEnvVars("GUARDIAN_ENABLED"),
+	}
+	FetchingProofTimeoutFlag = &cli.DurationFlag{
+		Name:    "fetching-proof-timeout",
+		Usage:   "Duration we will wait to fetching proof",
+		EnvVars: prefixEnvVars("FETCHING_PROOF_TIMEOUT"),
+		Value:   time.Hour * 2,
```
Comment (0xHansLee) [inline components/validator/flags/flags.go:122]: Why don't we change this to 4h?

**Review by 0xHansLee — APPROVED** [review]
lgtm

File: components/validator/flags/flags.go [hunk 3]
```diff
@@ -3,120 +3,123 @@ package flags
...
 	krpc "github.com/kroma-network/kroma/utils/service/rpc"
 	"github.com/kroma-network/kroma/utils/service/txmgr"
 )
 
-const envVarPrefix = "VALIDATOR"
+const EnvVarPrefix = "VALIDATOR"
+
+func prefixEnvVars(name string) []string {
+	return []string{EnvVarPrefix + "_" + name}
+}
```
Comment (seolaoh) [inline components/validator/flags/flags.go:19]: How about using utils function? Please apply same changes for the functions in node, batcher, txmgr flag!
```suggestion
func prefixEnvVar(name string) []string {
	return kservice.PrefixEnvVar(EnvVarPrefix, name)
}
```

File: components/validator/flags/flags.go [hunk 4]
```diff
@@ -3,120 +3,123 @@ package flags
...
-	kservice "github.com/kroma-network/kroma/utils/service"
 	klog "github.com/kroma-network/kroma/utils/service/log"
 	kmetrics "github.com/kroma-network/kroma/utils/service/metrics"
 	kpprof "github.com/kroma-network/kroma/utils/service/pprof"
 	krpc "github.com/kroma-network/kroma/utils/service/rpc"
 	"github.com/kroma-network/kroma/utils/service/txmgr"
 )
 
-const envVarPrefix = "VALIDATOR"
+const EnvVarPrefix = "VALIDATOR"
```
Comment (seolaoh) [inline components/validator/flags/flags.go:15]:
```suggestion
const envVarPrefix = "VALIDATOR"
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Comment by seolaoh** [comment]
Additionally, please apply PR conventions when naming the branch name next time :)

**Review by Pangssu — APPROVED** [review]

**Pull request merged**